### PR TITLE
Latest off-chain metadata additions including credit class json

### PR DESCRIPTION
**Pull request by wgwz**
## Description

Closes: https://github.com/regen-network/regen-registry/issues/780

Hi @blushi and @S4mmyb , I just wanted to open up a PR for Sam's branch so we can review. I actually think we should close this in favor of continuing the review process over in my PR #12 . I think Sam based his branch off of mine, so I've merged Sam's changes into my branch successfully and I also took care of the merge conflicts that both Sam's branch and my branch had with `main`. So I think it might make sense to just continue on in my PR and close this out. I just wanted to open it anyways in case I'm not understanding why it might be better separate.

The relevant files for Sam's changes are:

```
 jsonld/credit-batches/C01-verified-carbon-standard.json
 jsonld/credit-classes/C01-verified-carbon-standard-class.json
 jsonld/projects/C01-verified-carbon-standard-projects.json
```

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] manually tested (if applicable)

### CODE REVIEW

File: jsonld/credit-classes/C01-verified-carbon-standard-class.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "schema": "http://schema.org/",
+    "regen": "http://regen.network/",
+    "qudt": "http://qudt.org/schema/qudt",
+    "unit": "http://qudt.org/2.1/vocab/unit",
+    "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  "@type": "regen:CreditClass",
+  "regen:description": [
+      "This credit class provides a vehicle for nature based Verified Carbon Units (VCUs) to enter the blockchain space via issuance on Regen Ledger. It can be used by project developers, credit brokers, and other stakeholders interested in digitizing carbon credits issued by the VCS program to make them available in the emerging world of decentralized finance."
+  ],
```
Comment (S4mmyb) [inline jsonld/credit-classes/C01-verified-carbon-standard-class.json:12]:
```suggestion
  "regen:description": "",
```

File: jsonld/credit-classes/C01-verified-carbon-standard-class.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  "regen:sourceRegistry": {
+    "@type": "regen:Organization",
+    "schema:name": "Verra"
```
Comment (S4mmyb) [inline jsonld/credit-classes/C01-verified-carbon-standard-class.json:15]:
```suggestion
    "schema:name": ""
```

File: jsonld/credit-classes/C01-verified-carbon-standard-class.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  },
+  "regen:sectoralScope": {
+    "@type": "schema:ItemList",
+    "itemListElement": [
+      "Agriculture Forestry and Other Land Use (AFOLU)",
+      "Livestock and Manure Management"
+    ]
```
Comment (S4mmyb) [inline jsonld/credit-classes/C01-verified-carbon-standard-class.json:22]:
```suggestion
    "itemListElement": []
```

File: jsonld/credit-classes/C01-verified-carbon-standard-class.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  },
+  "regen:offsetGenerationMethod": {
+    "@type": "schema:ItemList",
+    "itemListElement": [
+      "Reduction",
+      "Removal"
+    ]
```
Comment (S4mmyb) [inline jsonld/credit-classes/C01-verified-carbon-standard-class.json:29]:
```suggestion
    "itemListElement": []
```

File: jsonld/credit-classes/C01-verified-carbon-standard-class.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    {
+      "@type": "regen:vcsMethodology",
+      "schema:name": "name",
+      "schema:URL": {
+        "@type": "schema:URL",
+        "value": "google.com"
+      }
+    }
+  ],
+  "regen:verificationMethod": "third-party verified"
```
Comment (S4mmyb) [inline jsonld/credit-classes/C01-verified-carbon-standard-class.json:49]:
```suggestion
  "regen:verificationMethod": ""
```

**Review by S4mmyb — COMMENTED** [review]
I'm for closing this and migrating over - I'll put int my comments there.

**Comment by blushi** [comment]
closing in favor of #12

**Pull request closed without merging**